commands/server.go

```diff
@@ -67,6 +67,10 @@ func server(cmd *cobra.Command, args []string) {
 		viper.Set("DisableLiveReload", disableLiveReload)
 	}
 
+	if viper.GetBool("watch") {
+    	serverWatch = true
+    }
```

Comment:
If you can `go fmt` this and move it below the next conditional (of no practical difference, but as it is now it is slightly confusing to read).